app/src/main/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabs.kt

```diff
@@ -63,14 +62,18 @@ import mozilla.components.browser.storage.sync.Tab as SyncTab
  */
 @OptIn(ExperimentalFoundationApi::class)
 @Composable
-fun SyncedTabsList(syncedTabs: List<SyncedTabsListItem>, onTabClick: (SyncTab) -> Unit) {
+fun SyncedTabsList(
+    syncedTabs: List<SyncedTabsListItem>,
+    taskContinuityEnabled: Boolean,
+    onTabClick: (SyncTab) -> Unit,
+) {
     val listState = rememberLazyListState()
 
     LazyColumn(
         modifier = Modifier.fillMaxSize(),
         state = listState,
     ) {
-        if (FeatureFlags.taskContinuityFeature) {
+        if (taskContinuityEnabled) {
```

Comment:
I think you can get away without passing `taskContinuityEnabled` because we have access to `components` in the @Composable. I think you should be call `components.settings` to access the feature flag.

Reply:
That's fair. I am okay as-is since this is temporary.